README.md

```diff
@@ -1,8 +1,8 @@
-# cloud.aws_ops roles/playbooks to demo Ansible on AWS
+# cloud.aws_ops roles/playbooks/rulebooks to demo Ansible on AWS
 
 This repository hosts the `cloud.aws_ops` Ansible Collection.
 
-The collection includes a variety of Ansible roles and playbooks to help automate the management of resources on AWS.
+The collection includes a variety of Ansible roles, playbooks, and rulebooks to help automate the management of resources on AWS.
 
 <!--start requires_ansible-->
 ## Ansible version compatibility
```

Comment:
This collection has been tested against following Ansible versions: **>=2.13.0**.